src/utility/ThreadUsingLambda.java

```diff
@@ -0,0 +1,19 @@
+package utility;
+
+public class ThreadUsingLambda {
+
+	public void lambdaExp() throws InterruptedException {
+		Thread t1 = new Thread(() -> {
```

Comment:
* Instead of passing lambda use method reference.
e.g 
  public void threadRunImpl() {
    System.out.println("Thread1"); 
  }
  Thread t1 = new Thread(this::threadRunImpl);

https://www.javatpoint.com/java-8-method-reference